app/src/main/java/com/example/mykotlincalculator21/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.mykotlincalculator21.mvp.model
+
+import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
+import com.example.mykotlincalculator21.utils.EMPTY_STRING
+
+class CalculatorModel : CalculatorContract.CalculatorModelContract {
+    override var firstOperand: String = EMPTY_STRING
```

Comment:
dont put properties on the contract. they shoud be private

Reply:
if he will use the variable for the native get and set I think it is okay, but as he is setting this variable with the method **saveNumber** the variable shouldn't be on the contract

Reply:
Done. Thx guys, now its fixed